app/models/manageiq/providers/redhat/infra_manager/host.rb

```diff
@@ -7,7 +7,7 @@ def provider_object(connection = nil)
 
   supports :quick_stats do
     unless ext_management_system.supports?(:quick_stats)
-      unsupported_reason_add(:quick_stats, 'RHV API version does not support quick_stats')
+      'RHV API version does not support quick_stats'
```

Comment:
Same here could this be `supports(:quick_stats) { ext_management_system.unsupported_reason(:quick_stats) }`